components/brave_ads/core/internal/ads_impl.cc

```diff
@@ -451,6 +463,8 @@ void AdsImpl::MigrateNotificationStateCallback(InitializeCallback callback,
 void AdsImpl::Start() {
   LogActiveStudies();
 
+  account_.SetWalletFrom(initial_wallet_.Get());
```

Comment:
Can we please add a comment to state this is a temporary fix until 1.54.x lands which has a more robust solution